musicname/zs_ffvii_mako_reactor_v6_2fix.cfg

```
@@ -0,0 +1,22 @@
+"music"
+{
+    "1w_1_ffvii_opening_theme_bombing_mission.mp3"              "FFVII Opening Theme - Bombing Mission"
+    "1w_2_ffvii_mako_reactor.mp3"                               "FFVII OST - Mako Reactor"
+    "1w_3_ffvii_fight_on!.mp3"                                  "FFVII OST - Fight On!"
+    "2w_1_ffvii_ac_black_water.mp3"                             "FFVII: Advent Children OST - Black Water"
+    "2w_2_ffvii_ac_battle_in_the_forgotten_city.mp3"            "FFVII: Advent Children OST - Battle in the Forgotten City"
+    "2w_3_ffvii_ac_jenova.mp3"                                  "FFVII: Advent Children OST - J-E-N-O-V-A"
+    "3w_1_ffvii_ac_beyond_the_wasteland.mp3"                    "FFVII: Advent Children OST - Beyond the Wasteland"
+    "3w_2_ffvii_ac_divinity_i.mp3"                              "FFVII: Advent Children OST - Divinity I"
+    "3w_3_ffvii_ac_one_winged_angel.mp3"                        "FFVII: Advent Children OST - One-Winged Angel"
+    "4w_1_pendulum_voodoo_people.mp3"                           "The Prodigy - Voodoo People (Pendulum Remix)"
+    "4w_2_pendulum_midnight_runner.mp3"                         "Pendulum - Midnight Runner"
+    "4w_3_the_qemists_stompbox.mp3"                             "The Qemists - Stompbox"
+    "4w_4_pendulum_the_other_side.mp3"                          "Pendulum - The Other Side"
+    "5w_1_pendulum_bloodsugar.mp3"                              "Pendulum - Blood Sugar"
+    "5w_2_pendulum_tarantula.mp3"                               "Pendulum - Tarantula (Pendulum vs fresh ft Spyda & Tenor Fly)"
```

Not entirely sure what you should do for displaying all the information for Tarantula.
```suggestion
    "5w_2_pendulum_tarantula.mp3"                               "Pendulum - Tarantula (Pendulum vs Fresh ft. Spyda & Tenor Fly)"
```

---

https://www.youtube.com/watch?v=NWsKQOJeL9o 
maybe should format it like this? i assume this is the original file being used 
`Pendulum & Fresh - Tarantula (ft. MC Spyda, Tenor Fly, MC Shabba)`